hudi-common/src/main/java/org/apache/hudi/common/util/HoodieRecordSizeEstimator.java

```diff
@@ -49,8 +48,9 @@ public long sizeEstimate(HoodieRecord<T> hoodieRecord) {
     // note the sizes and differences. A correct estimation in such cases is handled in
     /** {@link ExternalSpillableMap} **/
     long sizeOfRecord = ObjectSizeCalculator.getObjectSize(hoodieRecord);
-    long sizeOfSchema = ObjectSizeCalculator.getObjectSize(schema);
-    LOG.info("SizeOfRecord => " + sizeOfRecord + " SizeOfSchema => " + sizeOfSchema);
+    if (LOG.isDebugEnabled()) {
+      LOG.debug("SizeOfRecord => " + sizeOfRecord + " SizeOfSchema => " + sizeOfSchema);
+    }
```

Comment:
Would be still keep to `LOG.info` or Any reason to change to debug?

Reply:
Thanks for reviewing this. Use `LOG.info` will print each time when you put an element, `LOG.debug` is better.

Reply:
More, HoodieRecordSizeEstimator is mainly used in `ExternalSpillableMap`, `ExternalSpillableMap#put` has log it.
```
if (shouldEstimatePayloadSize && estimatedPayloadSize == 0) {
  // At first, use the sizeEstimate of a record being inserted into the spillable map.
  // Note, the converter may over estimate the size of a record in the JVM
  this.estimatedPayloadSize = keySizeEstimator.sizeEstimate(key) + valueSizeEstimator.sizeEstimate(value);
  LOG.info("Estimated Payload size => " + estimatedPayloadSize);
} 
```